service/api_logic/sports_logic.py

```diff
@@ -1,85 +1,37 @@
 from sqlalchemy import func
 from database.models import Sport, League, Country
-from dto.api_output import SportsLeagueOutput, SportsOutput
+from dto.api_output import SportsLeagueOutput, SportsOutput, ListResponseDTO
 from dto.pagination import Pagination
 from exept.handle_exeptions import handle_exceptions
-from service.api_logic.scripts import apply_filters
-from database.session import SessionLocal
 from logger.logger import Logger
+from service.api_logic.filter_manager.filter_manager_strategy import FilterManagerStrategy
 
-logger = Logger("logger", "all.log")
+class SportService:
+    def __init__(self, sports_dal, leagues_dal):
+        self._sports_dal = sports_dal
+        self._leagues_dal = leagues_dal
+        self._logger = Logger("logger", "all.log").logger
 
-session = SessionLocal()
+    def get_all_sports(self):
+        sports_query = self._sports_dal.get_base_query(Sport)
+        execute_query = self._sports_dal.execute_query(sports_query)
```

Comment:
does execute_query method returns another query?

if not - rename the variable properly